extensions/pongo2/activity.go

```diff
@@ -0,0 +1,129 @@
+package jinja2prompt
```

Comment:
@KulbhushanBhalerao As a best practice package name and module name must match. In this case module name is pongo2 but package name is jinja2prompt